spec/models/pawn_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Pawn, type: :model do
+  describe 'valid_move?' do
+    it 'should return true when moving one space forward' do
+      game = FactoryGirl.create(:game)
+      pawn = FactoryGirl.create(:pawn, x_position: 4, y_position: 2, game: game)
```

Comment:
I think in your factories the creation of the game is done for you. So could remove creating a game and setting it.